### PR TITLE
Change token request flow

### PR DESCRIPTION
**Pull request by Namiii**
Implemented the new Request builder flow for Token request

### CODE REVIEW

**Comment by pitiphong-p** [comment]
I think we should have the `v3` label

File: src/main/java/co/omise/Example.java [hunk 1]
```diff
@@ -334,27 +334,30 @@ void retrieveRefund() throws IOException, OmiseException, ClientException {
...
+        Request<Token> request = new Token.GetRequestBuilder("tokn_test_4xs9408a642a1htto8z").build();
+        Token token = client().sendRequest(request, Token.class);
         System.out.printf("token last digits: %s", token.getCard().getLastDigits());
     }
 
     void listTransactions() throws IOException, OmiseException, ClientException {
         Request<ScopedList<Transaction>> request = new Transaction.ListRequestBuilder().build();
-        ScopedList<Transaction> transactions = client().sendRequest(request, new TypeReference<ScopedList<Transaction>>() {});
+        ScopedList<Transaction> transactions = client().sendRequest(request, new TypeReference<ScopedList<Transaction>>() {
+        });
```
Comment (pitiphong-p) [inline src/main/java/co/omise/Example.java]: Can we remove the newline here?

Comment (Namiii) [reply src/main/java/co/omise/Example.java]: We can, but it formats it this every time I autoformat the code. It kind of bothers me as well. Fortunately it doesn't behave the same way in `Kotlin`

**Comment by Namiii** [comment]
`V3.0` label created and added

File: src/main/java/co/omise/models/Token.java [hunk 2]
```diff
@@ -40,13 +46,48 @@ public void setCard(Card card) {
         this.card = card;
     }
 
-    public static class Create extends Params {
+    /**
+     * The {@link RequestBuilder} class for for creating a Token.
```
Comment (nuxzero) [inline src/main/java/co/omise/models/Token.java]: `for for`? 😛

File: src/main/java/co/omise/models/Token.java [hunk 3]
```diff
@@ -40,13 +46,48 @@ public void setCard(Card card) {
...
+        }
+
+        public CreateRequestBuilder card(Card.Create card) {
             this.card = card;
             return this;
         }
     }
+
+    /**
+     * The {@link RequestBuilder} class for for retrieving a particular Token.
```
Comment (nuxzero) [inline src/main/java/co/omise/models/Token.java]: `for for`? 😛

File: src/main/java/co/omise/Example.java [hunk 4]
```diff
@@ -254,7 +253,8 @@ void destroyTransfer() throws IOException, OmiseException, ClientException {
     void listTransfers() throws IOException, OmiseException, ClientException {
         Request<ScopedList<Transfer>> request = new Transfer.ListRequestBuilder()
                 .build();
-        ScopedList<Transfer> transfers = client().sendRequest(request, new TypeReference<ScopedList<Transfer>>() {});
+        ScopedList<Transfer> transfers = client().sendRequest(request, new TypeReference<ScopedList<Transfer>>() {
+        });
```
Comment (nuxzero) [inline src/main/java/co/omise/Example.java]: Shall we keep this in the same line?

File: src/main/java/co/omise/models/Token.java [hunk 5]
```diff
@@ -15,7 +21,7 @@
  * Sending card data from server requires a valid PCI-DSS certification.
  * You can learn more about this in <a href="https://www.omise.co/security-best-practices">Security Best Practices</a>.
  * </p>
- *
+ * <p>
```
Comment (nuxzero) [inline src/main/java/co/omise/models/Token.java]: Missing close tag `</p>`?

File: src/test/java/co/omise/ClientTest.java [hunk 6]
```diff
@@ -26,18 +26,21 @@ public void testCreator() throws ClientException {
...
+                            .number("4242424242424242")
+                            .expiration(10, 2020)
+                            .securityCode("123"))
+                    .build();
+
+            new Client("pkey_test_123", "skey_test_123").sendRequest(request, Token.class);
         } catch (OmiseException e) {
             assertEquals("authentication_failure", e.getCode());
+        } catch (IOException e) {
+            e.printStackTrace();
```
Comment (nuxzero) [inline src/test/java/co/omise/ClientTest.java]: Could you use `failed()` here?

File: src/main/java/co/omise/models/Token.java (hunk 2 above)
Comment (Namiii) [reply src/main/java/co/omise/models/Token.java]: Why does this keep happening 😭

File: src/main/java/co/omise/models/Token.java (hunk 3 above)
Comment (Namiii) [reply src/main/java/co/omise/models/Token.java]: 👍

File: src/main/java/co/omise/Example.java (hunk 4 above)
Comment (Namiii) [reply src/main/java/co/omise/Example.java]: Autoformat keeps bringing it back down :/

File: src/test/java/co/omise/ClientTest.java (hunk 6 above)
Comment (Namiii) [reply src/test/java/co/omise/ClientTest.java]: As in `Assert.fail();`?

File: src/main/java/co/omise/models/Token.java (hunk 5 above)
Comment (Namiii) [reply src/main/java/co/omise/models/Token.java]: Will take care of it

**Comment by Namiii** [comment]
@nuxzero Changes made, please check

**Pull request merged**